src/main/java/teammates/logic/core/EmailGenerator.java

```diff
@@ -86,6 +88,87 @@
         return emails;
     }
     
+    /**
+     * Generates the summary of the feedback sessions (in which atleast one
+     * email has been sent to students) email for the given {@code courseId} for {@code student}
+     */
+    public EmailWrapper generateFeedbackSessionResendAllLinksEmail(String courseId,
+            StudentAttributes student) {
+        
+        CourseAttributes course = coursesLogic.getCourse(courseId);
+        
+        List<FeedbackSessionAttributes> sessions = new ArrayList<FeedbackSessionAttributes>();
+        List<FeedbackSessionAttributes> fsInCourse = fsDb.getFeedbackSessionsForCourse(courseId);
+        
+        for (FeedbackSessionAttributes fsa : fsInCourse) {
+            if (!fsa.isPrivateSession() && (fsa.isSentOpenEmail() || fsa.isSentPublishedEmail())) {
+                sessions.add(fsa);
+            }
+        }
+        
+        String submitUrl = null;
+        String reportUrl = null;
+        
+        StringBuffer emailBody = new StringBuffer(1000);
+        
+        emailBody.append("Hello " + student.name
+                        + ",<br> <br> An instructor has updated your email address in the course [Course name: "
+                        + course.getName() + "] [Course ID: " + course.getId() + "]. "
+                        + "Given below are the relevant links for the course, updated to work with your new email address "
+                        + student.email);
+        
```

Comment:
@chowyb should I create a new template for this also, instead of doing like this ?